### PR TITLE
initial account overview work

### PR DESCRIPTION
**Pull request by banukutlu**
Part of Issue #1 

## Additional change

* Added model specs for user and patron as part of #19

### CODE REVIEW

**Comment by cdmo** [comment]
Actually, never mind, I see that the user and patron specs are bound in with some of the other work here. I'm making my way through.

File: spec/models/patron_spec.rb [hunk 1]
```diff
@@ -3,5 +3,64 @@
 require 'rails_helper'
 
 RSpec.describe Patron, type: :model do
```
Comment (cdmo) [inline spec/models/patron_spec.rb]:
```suggestion
RSpec.describe Patron do
```
because of `config.infer_spec_type_from_file_location!`

Comment (cdmo) [reply spec/models/patron_spec.rb]: Fixed in #32

**Comment by cdmo** [comment]
Just to capture some of our conversation on Slack: there is probably some extraction of `Hold` and `Checkout` we can do to either `BibRecord` or perhaps a new model called `SirsiTransaction` (or something like that). Consider for a refactor later.

File: app/views/summaries/_summary.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="page-section">
+  <h3>Checkouts: <%= patron.checkouts.length %></h3>
+
+  <% count = patron.checkouts.select(&:recalled?).length %>
```
Comment (cdmo) [inline app/views/summaries/_summary.html.erb]: I wonder why this is in a view and not the controller (or a helper) 🤔

Comment (banukutlu) [reply app/views/summaries/_summary.html.erb]: hmm yes let's move it when we are working on the views

**Pull request merged**